src/System.Private.ServiceModel/tools/IISHostedWcfService/Web.config

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="utf-8"?>
+ <!--Copyright (c) Microsoft Corporation.  All Rights Reserved.-->
+<configuration>
+  <system.diagnostics>
+    <sources>
+      <source name="System.ServiceModel" switchValue="Information, ActivityTracing" propagateActivity="true">
+        <listeners>
+          <add name="traceListener" type="System.Diagnostics.XmlWriterTraceListener" initializeData="c:\log\Traces.svclog" />
```

Comment:
If the same trace file can be shared by multiple service instances, it will be fine. Yes, relative path will be nicer, eg., [path to root of repo]\..\WcfServiceLog\traces.svclog. This is not blocking.

During the setup we need to remember to grant corresponding app pool write permission to this folder.